ingestclient/bin/prep_ingest.py

```diff
@@ -0,0 +1,293 @@
+        type=config['channel']['type'], description=config['channel']['description'],
+        datatype=config['channel']['datatype'], sources=sources)
+    try:
+        channel = rmt.create_project(channel)
+    except:
+        channel = rmt.get_project(channel)
+    return channel
+
+
+def create_coord_frame(config, rmt):
+    """
+    Ensure coordinate frame exists.
+
+    Args:
+        config (dict):
+        rmt (BossRemote):
+
+    Returns:
+        (CoordinateFrameResource)
+    """
+    coord = CoordinateFrameResource(config['coordinate_frame']['name'],
+                                    config['coordinate_frame']['description'],
+                                    config['coordinate_frame']['x_start'],
+                                    config['coordinate_frame']['x_stop'],
+                                    config['coordinate_frame']['y_start'],
+                                    config['coordinate_frame']['y_stop'],
+                                    config['coordinate_frame']['z_start'],
+                                    config['coordinate_frame']['z_stop'],
+                                    config['coordinate_frame']['voxel_size_x'],
+                                    config['coordinate_frame']['voxel_size_y'],
+                                    config['coordinate_frame']['voxel_size_z'])
+    try:
+        coord = rmt.create_project(coord)
+    except:
+        coord = rmt.get_project(coord)
+    return coord
+
+
+def add_gov_team_permissions(rmt, collection, experiment, channel):
+    """
+    Give read permission to the government team.
+
+    Args:
+        rmt (BossRemote):
+        collection (CollectionResource):
+        experiment (ExperimentResource):
+        channel (ChannelResource):
+    """
+
+    try:
+        rmt.add_permissions(GOV_TEAM, collection, ['read'])
+    except:
+        print('Failed to automatically add {} group to collection: {}'.format(GOV_TEAM, collection.name))
+
+    try:
+        rmt.add_permissions(GOV_TEAM, experiment, ['read'])
+    except:
+        print('Failed to automatically add {} group to experiment: {}'.format(GOV_TEAM, experiment.name))
+    try:
+        rmt.add_permissions(GOV_TEAM, channel, ['read'])
+    except:
+        print('Failed to automatically add {} group to channel: {}'.format(GOV_TEAM, channel.name))
+
+
+def main(args):
+    """
+    Main entry point of script.
+
+    Args:
+        args (Namespace): Parsed command line arguments.
+    """
+    rmt = BossRemote(args.intern_cfg)
+
+    # Load Configuration File
+    with open(args.config_file, 'rt') as cfg:
+        config = json.load(cfg)
+
+    collection = create_collection(config, rmt)
+    coord = create_coord_frame(config, rmt)
+    experiment = create_experiment(config, rmt, collection, coord)
+    channel = create_channel(config, rmt, collection, experiment)
+
+    add_gov_team_permissions(rmt, collection, experiment, channel)
```

Comment:
I checked with Will.  Lets remove this for now, we don't want to be adding gov_team until this tell us to do so.